### ISSUE TITLE
Reduced circulating tumor necrosis factor alpha concentration

### ISSUE THREAD

**Issue by caripark** [obophenotype/human-phenotype-ontology#11720]
**Preferred term label:**
reduced circulating tumor necrosis factor alpha concentration

**Synonyms**
reduced circulating TNF alpha concentration

**Definition (free text, please give PubMed ID)**
An abnormally decreased concentration of tumor necrosis factor alpha in the blood circulation. (PMID: 24120359)

**Parent term (use hpo.jax.org/app)**
Abnormal circulating cytokine concentration (https://hpo.jax.org/browse/term/HP:0011112) OR new term: Abnormal circulating tumor necrosis factor alpha concentration

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-2346-5201

**Comment by pnrobinson** [comment]
@caripark The baseline level of tumor necrosis factor alpha in the blood circulation is very low, and there is no accepted "lower limit of normal", i.e., a result of zero would not be considered abnormal. An HPO term needs to be applicable to individual patients (rather than cohorts, were the mean value of TNFa might be lower for a given disease): that is, we would need to have a normal range such as 3.5-5mg/dl where it is clear that a value is too high or too low independent of the clinical circumstance. 
I am not sure but there may be general guidelines for a lower than expected TNFa response to an infection. I am not sure if this is research only, since I could not find normal ranges. Please reopen or create a new issue if it would be appropriate to make a term here.
